### DISCUSSION TITLE
[Bug]: Heltec Wireless Tracker 1.1 can’t  entering flashmode with bme280 on pin 45 46

### DISCUSSION THREAD

**Discussion by Taxom** [meshtastic/firmware#5291] [Troubleshooting]
### Category

Other

### Hardware

Heltec Wireless Tracker

### Firmware Version

2.5.6.d55c08d +

### Description

I encountered an issue with entering flash mode on the Heltec Wireless Tracker. When an external BME280 sensor is connected to GPIO 45 and 46, it’s impossible to enter flash mode. Holding the USER button while powering on the device doesn’t work, and the regular program loads instead of going into flash mode. The USER button itself functions correctly.
I was only able to enter flash mode after physically disconnecting the BME280 from pins 45 and 46. Has anyone else faced a similar issue, or does anyone know of a workaround to avoid disconnecting the wires every time I need to update the firmware?

Thanks!

**Comment by b8b8** [comment]
So this interesting to me because i think it was (you?) mentioned to me discord. I hooked up a BME680 and it worked no problem.
I wonder if its only a problem with the BME280.
![image](https://github.com/user-attachments/assets/5b24ea94-32dd-4b82-812f-0496843278a0)

**Comment by lyusupov** [comment]
It does not look like a smart decision to choose ESP32-S3 "strapping" pins for the I2C purpose:

<img width="521" alt="image" src="https://github.com/user-attachments/assets/7c992fc6-de6b-4ef8-930b-a72e754f462f">

<br>
<br>

<img width="640" alt="image" src="https://github.com/user-attachments/assets/5a8afd65-490f-4bac-898b-b6b48563b0d6">

<br>
<br>

<img width="761" alt="image" src="https://github.com/user-attachments/assets/8c874932-c87a-40c4-9e61-34d4feeda6a7">

<br>
<br>

One known good example is [**SoftRF project**](https://github.com/lyusupov/SoftRF)
It uses spare pins 6 and 7 with no issues:

<img width="650" alt="image" src="https://github.com/user-attachments/assets/9face57c-2517-498b-a3e1-8cf3dab96a25">

**Comment by Uolsen** [comment]
Hi guys. I encountered the same issue. Is somebody working on this? It should not be closed because it is quite a bug. I almost broke down until I figured out why flashing was not working for me.
